src/data/InstanceDhisRepository.ts

```diff
@@ -689,7 +689,6 @@ export class InstanceDhisRepository implements InstanceRepository {
                                   latitude: geometry.coordinates[1]?.toString() ?? "",
                               }
                             : coordinate,
-                        geometry: geometry,
```

Comment:
Ok, `geometry`  was not in the type. 

About the trackedEntityInstance -> trackedEntity change, the type confirms is the right prop, but I am surprised this was working before, and the PR intends to change another problem. 

(But I don't have the full picture)